conf/application.conf

```diff
@@ -86,7 +89,7 @@ metrics {
   durationUnit = SECONDS
   showSamples = true
   jvm = true
-  enabled = true
+  enabled = false
```

Comment:
Was this to make tests pass? If so it might be best to revert this and set it in` test.application.conf` instead, particularly since platops now suggest we remove these values from our local configurations